docker_image_build.sh

```diff
@@ -0,0 +1 @@
+docker build -t protocol-splitter-cont --no-cache .
```

Comment:
There is nothing wrong in this file (or the other one), but philosophically both of these are instructions that IMO could be in a README like this:

```
How to run in Docker
----------------------------

'''
$ docker run --name prot_split --rm -dt --device /dev/ttyS7 protocol-splitter-cont 127.0.0.1 /dev/ttyS7
'''

How to build Docker image
------------------------------------

'''
$ docker build -t protocol-splitter-cont --no-cache .
'''
```

Rationale: usually these scripts are not used/required in production or the CI (the `.github/workflows/..` file already has its own explicit way of doing this). In production there's also an explicit spell for running it: https://github.com/tiiuae/fog_hyper/blob/37b43c44ee110491bc4571e3ce68dd944d1964f5/pkg/manifest/manifest.go#L73
=> these scripts serve the occasional developer/user who should be fine copy-pasting the advice from readme and making their own scripts if they have repetitive / exotic needs

But this is just an opinion, there is nothing wrong in these being as scripts, no need to spend time "fixing" even if you agree. Just a thought to think about in the future :)

Reply:
Right, those scripts were for my own testing only, they are accidentally taken in. Definitely not needed to be added here